.vscode/settings.json

```diff
@@ -117,5 +117,9 @@
         "editor.defaultFormatter": "ms-python.autopep8",
         "editor.formatOnSave": true,
         "editor.formatOnSaveMode": "modifications"
-    }
+    },
+    "python.testing.pytestArgs": [
+        "."
+    ],
```

Comment:
What's this solving?  It should already be handled in `setup.cfg`:

```dosini
[tool:pytest]
...
testpaths = mlos_core mlos_bench
```

Reply:
oops, I just did not notice that stuff got committed. I will roll back this change now